### ISSUE TITLE
/etc/sysctl will overwrite other system wide modifications

### ISSUE THREAD

**Issue by andrewhowdencom** [apolloclark/ansible-role-osquery#1]
Hola,

Just reading through and noticed that you're templating /etc/sysctl.conf. You'll overwrite other system wide modification shere; that precludes it from our use. 

<3

**Comment by andrewhowdencom** [comment]
On Debian Stretch there's `/etc/sysctl.d`; might be better for your purposes?

**Comment by Shackelford-Arden** [comment]
@andrewhowdencom I'd bet you'd be better off taking the fine work of @apolloclark and modifying it for your purposes. You could use the linefile module instead of a template to add the necessary bits to your systems.

**Comment by apolloclark** [comment]
Yup, I'll modify the role tonight, to use linefile instead of over writing the whole thing. Thanks! :)
